stock_picking_auto_create_package/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+* Joshua Lauer <joshua.lauer@raumschmiede.de>
+* Michael Tietz (MT Software) <mtietz@mt-software.de>
```

Comment:
@jbaudoux Could you add me too as the original idea to put that functionnality in a dedicated module come from https://github.com/OCA/delivery-carrier/pull/642

Reply:
sure, done